src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 7) {
+    return (days * 40) - 50;
+  }
+
+  if (days >= 3) {
+    return (days * 40) - 20;
+  }
```

Comment:
Move `days * 40` and other magic numbers like `20`, `50 `and `40` to variables.

Also last if statement is redundant because you have return inside previous if's